tests/test_kex.py

```diff
@@ -23,9 +23,11 @@
 from binascii import hexlify, unhexlify
 import os
 import unittest
+import pytest
 
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.asymmetric import x25519
```

Comment:
this import also needs fallback to `x25519 = None`

Reply:
Fixed